src/Containers/SquareTable/_ChairRow.tsx

```diff
@@ -159,9 +159,10 @@ interface ISideChairRow {
 const SideChairRow = styled.div<ISideChairRow>`
     display: flex;
     flex-wrap: wrap;
-    margin-right: ${({ relativeSize }) => -15 * relativeSize}px;
-    margin-left: ${({ relativeSize }) => -15 * relativeSize}px;
-    height: ${({ relativeSize }) => 20 * relativeSize}rem;
+    ${({ relativeSize }) =>
+        `margin-right: ${-15 * relativeSize}px;
```

Comment:
This -15 is a magic number what does it mean?
WHITESPACE_BETWEEN_CHAIR_AND_TABLE?